lit/docs/jobs/steps.lit

```diff
@@ -679,6 +679,23 @@ by configuring \reference{schema.step.on_failure} or
         in multiple files, the value from a file that is passed later in the
         list will override the values from files earlier in the list.
       }
+
+      \optional-attribute{team}{identifier}{
+        By default the \code{set_pipeline} step sets pipelines for the
+        pipeline's \reference{teams}{team}.
+
+        The \code{team} attribute can be used to specify a specific team.
```

Comment:
I wonder if we could frame this as "the default value for this field is the team that is running the build"?